packages/testing/ts-sinon/src/mocks.ts

```diff
@@ -0,0 +1,116 @@
+import * as sinon from 'sinon';
+import { SinonStub } from 'sinon';
+
+type DeepPartial<T> = {
```

Comment:
Were the types/helpers copied from `ts-jest`? If so i would recommend having a common package to re-usable the code/types between code

Reply:
So you want to remove the types from `ts-jest` and `ts-sinon` and put them in `@golevelup@tests-types`?

Reply:
@omermorad If the implementation is the same but what differs is the underlying types/package we could extract it to a common place. I don't think it should be a blocker though but could be a follow up

Reply:
So, shall we merge this pull request and open a new issue for further discussion?

Reply:
@omermorad Sure, I'm down for it :)